packages/xod-client-electron/src/app/main.js

```diff
@@ -53,6 +53,8 @@ function createWindow() {
     minHeight: 600,
     title: 'XOD IDE',
     show: false,
+    // this is required for subpixel antialiasing to work
+    backgroundColor: '#FFF',
```

Comment:
In the PR description, please, refer to #635 (we’re going to close it) so that we’d have audit trail